presentation/chapters/es-ES/unsafe.chapter

```diff
@@ -0,0 +1,65 @@
+## Unsafe Rust
+
+El sistema de tipo de Rust provee muchas garantías, pero en ocasiones esto conlleva que soluciones específicas sean difíciles o imposibles.
+
+Por esta razón, Rust tiene el concepto de "código inseguro".
+
+---
+
+El código inseguro tiene permitido:
+-   acceder memoria libremente
+-   utilizar apuntadores crudos (raw pointers)
+-   llamar a funciones externas
+-   declarar valores `Send` y `Sync`
+-   escribir a variables globales no sincronizadas
+
+---
+
+No es seguro:
+
+-   conversión a apuntadores crudos
```

Comment:
This was hard to translate :) It's a good idea to use `sin procesar` as translation of `raw`.